boards/arm/nucleo_f411re/Kconfig.defconfig

```diff
@@ -16,4 +16,11 @@ config SPI_STM32_INTERRUPT
 	default y
 	depends on SPI
 
+if I2S
+
+config I2S_1
+	default y
+
+endif # I2S
```

Comment:
Please remove this.

Reply:
> Please remove this.

@erwango 
Removed, re-tested, re-pushed.
Thanks again!